doc/themes/the-things-stack/layouts/shortcodes/udp-pf.html

```diff
@@ -0,0 +1 @@
+Semtech UDP Packet Forwarder
```

Comment:
Is this worth a short code? This hasn't been changed in six years and I doubt it ever will

Reply:
what's the downside? i thought we might want to change how we refer to it

Reply:
Mostly wondering about the need for it. It's a sliding edge because before you know it, we have shortcodes for all product names